virtcontainers/fc.go

```diff
@@ -242,17 +289,53 @@ func (fc *firecracker) fcInit(timeout int) error {
 	span, _ := fc.trace("fcInit")
 	defer span.Finish()
 
-	args := []string{"--api-sock", fc.socketPath}
+	if fc.config.JailerPath != "" {
+		fc.jailed = true
+	}
+
+	// Fetch sandbox network to be able to access it from the sandbox structure.
+	var networkNS NetworkNamespace
+	if err := fc.store.Load(store.Network, &networkNS); err == nil {
+		if networkNS.NetNsPath == "" {
+			fc.Logger().WithField("NETWORK NAMESPACE NULL", networkNS).Error()
+		}
+		fc.netNSPath = networkNS.NetNsPath
+	}
+
+	var args []string
+	var cmd *exec.Cmd
+
+	if fc.jailed {
+		args = []string{
+			"--id", fc.id,
+			"--node", "0", //FIXME: Comprehend NUMA topology or explicit ignore
```

Comment:
For now, don't set it unless the local allocator (kubelet etc.) tells us which NUMA node to use?

Reply:
I see. Let's have it for now then.